magma/fromverilog.py

```diff
@@ -81,15 +87,11 @@ def get_type(io, type_map):
 def ParseVerilogModule(node, type_map):
     args = []
     ports = []
-    for port in node.portlist.ports:
-        if isinstance(port, Ioport):
-            io = port.first
-            args.append(io.name)
-            args.append(get_type(io, type_map))
-        elif isinstance(port, Port):
-            ports.append(port.name)
-        else:
-            raise NotImplementedError(type(port))
```

Comment:
Was this port related code supposed to be removed? My suspicion as to why the tests are failing are because we still need to have `ports` and `args` populated

Reply:
oops.